src/test/java/org/cactoos/http/io/BlockingSocketServerTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
```

Comment:
@fabriciofx I can't do that, qulice will complain because it checked it against the `LICENSE.txt` of the repositoty, ARC should change that all at once usually, maybe open a ticket instead?

Reply:
@victornoel please, open a ticket about it.

Reply:
@fabriciofx I did, #94, thx for letting me have the reward ;)